index.html

```diff
@@ -0,0 +1,205 @@
+            </li><hr>
+            <li>
+              <a href="#">Contact</a>
+            </li><hr>
+          </ul>
+        </section>
+        <section class="mobile-navigation">
+          <ul>
+            <li>
+              <a href="#"><img src="images/mobile_menu 1.svg" id="menu" alt="menu"></a>
+            </li>
+            <li>
+              <a href="index.html"><img src="images/fmcde-round-logo.png" id="logo" alt="logo"></a>
+            </li>
+          </ul>
+        </section>
+        <section class="desktop-nav1">
+          <ul>
+            <li>
+              <a href="#">
+                <img src="images/facebook.svg" alt="facebook">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/twitter.svg" alt="twitter">
+              </a>
+            </li>
+            <li>
+              <a href="#">English</a>
+            </li>
+            <li>
+              <a href="#">MyPage</a>
+            </li>
+            <li>
+              <a href="#">Logout</a>
+            </li>
+          </ul>
+        </section>
+        <section class="desktop-nav2">
+          <a href="index.html">
+            <img src="images/fmcde-main-logo-trans.png" id="main-logo" alt="logo">
+          </a>
+          <ul>
+            <li>
+              <a href="index.html">Home</a>
+            </li>
+            <li>
+              <a href="about.html">About</a>
+            </li>
+            <li>
+              <a href="#">Initiatives</a>
+            </li>
+            <li>
+              <a href="#">Resources</a>
+            </li>
+            <li>
+              <a href="#">Contact</a>
+            </li>
+            <li>
+              <a href="#" class="news">News</a>
+            </li>
+          </ul>
+        </section>
+      </nav>
+    </header>
+    <main>
+      <section class="main-sec">
+        <div class="title-sec">
+          <h4 class="orange">FEDERAL MINISTRY OF</h4>
+          <h1 class="orange">COMMUNICATIONS AND DIGITAL ECONOMY</h1>
+        </div>
+        <section class="mission-vision">
+          <h3>Vision</h3>
+          <p>
+            To create a robust digital economy that enhances security, increases 
+            transparency, and diversifies the Nigerian economy.
+          </p>
+          <h3>Mission</h3>
+          <p>We leverage Digital Technologies for National Economic Development.</p>
+        </section>
+      </section>
+      <section class="program">
+        <section class="sec-tit">
+          <h2 class="gray">Our Agencies</h2>
+          <hr class="hr">
+        </section>
+        <ul class="agencies">
+          <li class="agency1">
+            <img src="images/ncc.png" alt="NCC">
+            <h3 class="orange">NCC</h3>
+            <p>
+              Provides exposure to the cadets in a wide range of activities., with a distinct 
+              emphasis on Social Services, Discipline and Adventure Training.
```

Comment:
- [ ] Good work up to this point but there are some aspects you can improve on. Your agency cards on both the desktop and mobile versions appear to be different from the original design. Kindly improve on them to match the original design.

![image](https://user-images.githubusercontent.com/98453979/235203398-b36e12c6-5a20-46bf-9186-b09ed2d921c3.png)
 - [ ] On the desktop version, increase the general padding around the cards
 
![image](https://user-images.githubusercontent.com/98453979/235203755-bfd066e3-19f7-4149-b306-872bdc2df535.png)